src/components/Footer/styles.scss

```
@@ -9,3 +9,9 @@
     height: rem(72px);
   }
 }
+
+.footer-item {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: normal;
+}
```

How about approach it sass style?

```suggestion
@import '../../styles/main.scss';

.footer {
  height: rem(32px);
  background-color: $secondary;
  
  @include flex-center();
  @include respond-above(md) {
     height: rem(72px);
   }
   
  &__item {
    font-size: 16px;
    font-weight: 500;
    line-height: normal;
  }
}
```

---

Source: https://css-tricks.com/sass-style-guide/